platform/bootstrap/bootstrap.yml

```diff
@@ -4,24 +4,97 @@ Description: Docker Swarm cluster deployed with InfraKit
 Mappings:
   AMI:
     # Ubuntu 16.04 HVM
-    eu-central-1:
-      Ubuntu: ami-1b4d9e74
+    ap-southeast-2:
+      Ubuntu: ami-92e8e6f1
     eu-west-1:
       Ubuntu: ami-b5a893d3
-    us-east-1:
-      Ubuntu: ami-e4139df2
-    us-west-1:
-      Ubuntu: ami-30476250
+    us-east-2:
+      Ubuntu: ami-33ab8f56
     us-west-2:
       Ubuntu: ami-17ba2a77
   VpcCidrs:
     subnet1:
-      cidr: 192.168.2.0/24
+      cidr: 192.168.0.0/24
+    subnet2:
+      cidr: 192.168.16.0/24
+    subnet3:
+      cidr: 192.168.32.0/24
     vpc:
       cidr: 192.168.0.0/16
 
+Metadata:
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+    - Label:
+        default: Swarm Size
+      Parameters:
+      - ManagerSize
+      - WorkerSize
+    - Label:
+        default: Swarm Properties
+      Parameters:
+      - KeyName
+    - Label:
+        default: Swarm Manager Properties
+      Parameters:
+      - ManagerInstanceType
+      - ManagerDiskSize
+      - ManagerDiskType
+    - Label:
+        default: Swarm Worker Properties
+      Parameters:
+      - WorkerInstanceType
+    - Label:
+        default: InfraKit Configuration
+      Parameters:
+      - InfraKitConfigurationBaseURL
+    - Label:
+        default: Docker Configuration
+      Parameters:
+      - AufsVolumeSize
+    ParameterLabels:
+      ManagerSize:
+        default: Number of Swarm managers?
+      ManagerDiskSize:
+        default: Manager ephemeral storage volume size?
+      ManagerDiskType:
+        default: Manager ephemeral storage volume type
+      WorkerSize:
+        default: Number of Swarm workers?
+      KeyName:
+        default: Which SSH key to use?
+      EnableCloudWatchLogs:
```

Comment:
we should remove the cloudwatch settings for now, we should rely on a more generic monitoring system